doc/template_definition.md

```diff
@@ -290,3 +501,23 @@ is transformed into:
     java.lang.System.out.println(1);
 }
 ```
+## Inlining with PatternBuilder
+The template code in spoon templates made by `PatternBuilder` is never inlined automatically.
+But you can mark code to be inlined this way:
+```java
+Pattern t = PatternBuilder.create(...select template model...)
+	//...configure parameters...
+	configureLiveStatements(ls -> 
```

Comment:
so `configureLiveStatements` is used only for `inlining`? Shouldn't it be renamed then?

Reply:
yes, it is for inlining. This PR really needs a lot of renaming and I am open to any better names.

Reply:
why not `configureInlineStatements` then?

Reply:
ok, I will rename all "live" to "inline"